editor/py/serve.py

```diff
@@ -109,6 +144,19 @@ def show_reaction(file_name, index):
       'reaction.html', file_name=file_name, index=index)
 
 
+@app.route('/reaction/download', methods=['POST'])
+def download_reaction():
```

Comment:
What is the purpose of this endpoint? Why would a user want to perform this transformation?

Reply:
A user might want to download a single reaction pbtxt to use as one of the inputs for `enumerate_dataset`, either on the webform or using the Python script.

Reply:
Aren't the reactions stored as pbtxt anyway?

Can we skip the intermediate binary stage? Just serve static content?

Reply:
Reactions are only stored as submessages in dataset pbtxt files, not on their own. Having the reaction proto be serialized and send in a POST also allows users to download reaction.pbtxt files based on the exact state of the form without having to save/commit to the dataset.